string task_id
string loader_id
string unloader_id
float64[2] pickup_location
float64[2] dropoff_location
builtin_interfaces/Time created_at
